client-misk-jooq/src/test/kotlin/app/cash/backfila/client/misk/setup/JooqSqlLogger.kt

```diff
@@ -0,0 +1,33 @@
+package app.cash.backfila.client.misk.setup
```

Comment:
Is a package name called setup a Jooq thing?

I think I'd normally see it in a config package. Something like app.cash.backfila.client.misk.config

Reply:
no its just a name i chose. Made this config.